app/scripts/controllers/main.js

```diff
@@ -57,18 +61,23 @@ angular.module('moodCatApp')
     $scope.selectRoom = function selectRoom(room) {
       $scope.activeRoom = room;
       $scope.loadSong(room.song.soundCloudId);
-    }
 
-    /** CHAT **/
+      //Fetch chat
```

Comment:
This is an example of a comment that should have been unnecessary if the code was properly refactored :wink: I suggest to extract lines 67 to 78 to a method/function `fetchChat()` and omit the comment.